docs/checkbox/readme.md

```diff
@@ -58,15 +58,26 @@ export class BasicDemo extends React.Component<{}, {value: boolean}> {
 
 ## Style API
 
+### Subcomponents (pseudo-elements)
+
+| selector            | description                              |
+| ------------------- | ---------------------------------------- |
+| ::box               | Frame of the checkbox                    |
+| ::icon              | Positioning and size for both icons      |
```

Comment:
do any of these internal parts have their own states?

Reply:
No, not as the component is built right now